cfme/tests/containers/test_containers_properties.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+# the test performs field validation in Properties table
+# Polarion test 9911
+# Polarion test 9877
+# Polarion test 9867
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.pod import Pod
+from cfme.containers.route import Route
+from cfme.containers.project import Project
+from utils import testgen
+from utils.version import current_version
+from cfme.web_ui import CheckboxTable
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+# container pods Properties table field verification
+@pytest.mark.parametrize('rel',
+                         ['Name',
+                          'Phase',
+                          'Creation timestamp',
+                          'Resource version',
+                          'Restart policy',
+                          'DNS Policy',
+                          'IP Address'])
+def test_pods_properties_rel(provider, rel):
+    sel.force_navigate('containers_pods')
+    list_tbl_pod = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+    ui_pods = [r.name.text for r in list_tbl_pod.rows()]
+    mgmt_objs = provider.mgmt.list_container_group()  # run only if table is not empty
+    validate_str = False
+
+    if ui_pods:
+        # verify that mgmt pods exist in ui listed pods
+        assert set(ui_pods).issubset(
+            [obj.name for obj in mgmt_objs]), 'Missing objects'
+
+    for name in ui_pods:
+        obj = Pod(name, provider)
+        val = obj.get_detail('Properties', rel)
```

Comment:
Check if Pod has SummaryMixin. If not, you can add it to it as long as Pod implements load_details()

Then you could write something like getattr(obj.summary.provider, rel).text_value (or .value if you want to have it converted to number). It should handle it gracefully.